src/renderer/component/fileDetails/view.jsx

```diff
@@ -1,77 +1,128 @@
 // @flow
-import * as React from 'react';
+import type { Claim, Metadata } from 'types/claim';
+import type { FileInfo } from 'types/file_info';
+import React, { Fragment, PureComponent } from 'react';
+import { Lbryio } from 'lbryinc';
 import MarkdownPreview from 'component/common/markdown-preview';
 import Button from 'component/button';
 import path from 'path';
-import type { Claim } from 'types/claim';
+import Expandable from 'component/expandable';
 
 type Props = {
   claim: Claim,
-  fileInfo: {
-    download_path: string,
-  },
-  metadata: {
-    description: string,
-    language: string,
-    license: string,
-  },
+  fileInfo: FileInfo,
+  metadata: Metadata,
   openFolder: string => void,
   contentType: string,
+  clickCommentButton: () => void,
+  showSnackBar: string => void,
+  hasClickedComment: boolean,
+  user: ?any,
 };
 
-const FileDetails = (props: Props) => {
-  const { claim, contentType, fileInfo, metadata, openFolder } = props;
+class FileDetails extends PureComponent<Props> {
+  constructor() {
+    super();
+    (this: any).handleCommentClick = this.handleCommentClick.bind(this);
+  }
 
-  if (!claim || !metadata) {
-    return (
-      <div className="card__content">
-        <span className="empty">{__('Empty claim or metadata info.')}</span>
-      </div>
-    );
+  handleCommentClick() {
+    const { clickCommentButton, showSnackBar } = this.props;
+
+    clickCommentButton();
+    Lbryio.call('user_tag', 'edit', { add: 'comments-waitlist' });
+    showSnackBar(__('Thanks! This feature is so beta, all we had time for was this button.'));
   }
 
-  const { description, language, license } = metadata;
+  render() {
+    const {
+      claim,
+      contentType,
+      fileInfo,
+      metadata,
+      openFolder,
+      hasClickedComment,
+      user,
+    } = this.props;
 
-  const mediaType = contentType || 'unknown';
-  const downloadPath = fileInfo ? path.normalize(fileInfo.download_path) : null;
+    if (!claim || !metadata) {
+      return (
+        <div className="card__content">
+          <span className="empty">{__('Empty claim or metadata info.')}</span>
+        </div>
+      );
+    }
+
+    const { description, language, license } = metadata;
 
-  return (
-    <React.Fragment>
-      {description && (
-        <React.Fragment>
-          <div className="card__subtext-title">About</div>
+    const mediaType = contentType || 'unknown';
```

Comment:
Should `'unknown'` be a shared constant?

Reply:
Probably. Although I'm not sure where else it's used.